Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/Helpers/ManageBericht.cs

```diff
@@ -235,6 +245,17 @@ public static async Task<bool> AreSkillsVisibleByNameAsync(this IPage page, ILoc
 
             return true;
         }
+        public static async Task WaitForPageAndSpinnerAsync(this IPage page)
+        {
+            // Wait for the page to load completely
+            await page.WaitForLoadStateAsync(LoadState.DOMContentLoaded);
```

Comment:
have you tried it without this line? i think this is for the whole page, which was already loaded. we're just waiting for a part of it to refresh

Reply:
i tested it and I removed the line